ui/src/app/kvm/views/KVMList/KVMListTable/KVMListTable.test.tsx

```diff
@@ -12,17 +27,17 @@ const mockStore = configureStore();
 describe("KVMListTable", () => {
   let initialState;
   beforeEach(() => {
-    initialState = {
-      controller: {
+    initialState = rootStateFactory({
+      controller: controllerStateFactory({
         loaded: true,
         loading: false,
         items: [],
```

Comment:
Feel free to remove any of these model attributes that aren't required for the test (in the future).

Reply:
Oh yeah I totally meant to do that but got lost in the copy+pastey sauce